src/main/java/net/fabricmc/loader/entrypoint/minecraft/EntrypointPatchHook.java

```diff
@@ -391,6 +392,9 @@ public void process(FabricLauncher launcher, Consumer<ClassNode> classEmitter) {
 					finishEntrypoint(type, it);
 				}
 				patched = true;
+			} else if (isRunnable || entrypoint.contains("com.mojang.rubydung.RubyDung")) {
+				System.out.println("Trying as Runnable");
```

Comment:
you left debug code

Reply:
I'll remove it.

Reply:
Gone and replaced with a generic comment stating that RD should not need a real patch